### ISSUE TITLE
WOTLK Classic - getCooldownFunc

### ISSUE THREAD

**Issue by Nenos-Hu** [mitchnull/CooldownToGo#2]
Hello,

Error pops on WOTLK classic:

```10398x CooldownToGo/CooldownToGo.lua:546: attempt to call local 'getCooldownFunc' (a nil value)
[string "@CooldownToGo/CooldownToGo.lua"]:546: in function `showCooldown'
[string "@CooldownToGo/CooldownToGo.lua"]:638: in function `checkItemCooldown'
[string "@CooldownToGo/CooldownToGo.lua"]:571: in function <CooldownToGo/CooldownToGo.lua:565>
[string "=(tail call)"]: ?
[string "=[C]"]: ?
[string "=[C]"]: in function `UseAction'
[string "@FrameXML/SecureTemplates.lua"]:343: in function `handler'
[string "@FrameXML/SecureTemplates.lua"]:625: in function <FrameXML/SecureTemplates.lua:569>
[string "=[C]"]: ?
[string "@FrameXML/SecureHandlers.lua"]:266: in function <FrameXML/SecureHandlers.lua:263>
[string "=[C]"]: ?
[string "@FrameXML/SecureHandlers.lua"]:296: in function <FrameXML/SecureHandlers.lua:279>
[string "=(tail call)"]: ?

Locals:
self = <table> {
 SetDefaultModuleLibraries = <function> defined @Attune/Libs/AceAddon-3.0/AceAddon-3.0.lua:363
 unlock = <function> defined @CooldownToGo/CooldownToGo.lua:361
 EnableModule = <function> defined @Attune/Libs/AceAddon-3.0/AceAddon-3.0.lua:328
 modules = <table> {
 }
 openConfigDialog = <function> defined @CooldownToGo_Options/Options.lua:240
 updateStamps = <function> defined @CooldownToGo/CooldownToGo.lua:506
 SecureHookScript = <function> defined @Bartender4/libs/AceHook-3.0/AceHook-3.0.lua:409
 icon = CDTGIcon {
 }
 iconTexture = CDTGIcon {
 }
 RawHookScript = <function> defined @Bartender4/libs/AceHook-3.0/AceHook-3.0.lua:394
 setupLDB = <function> defined @CooldownToGo/CooldownToGo.lua:756
 updateLayout = <function> defined @CooldownToGo/CooldownToGo.lua:192
 SetDefaultModulePrototype = <function> defined @Attune/Libs/AceAddon-3.0/AceAddon-3.0.lua:405
 OptionsAppName = "CooldownToGo_Options"
 defaultModuleState = false
 IsEnabled = <function> defined @Attune/Libs/AceAddon-3.0/AceAddon-3.0.lua:447
 DisableModule = <function> defined @Attune/Libs/AceAddon-3.0/AceAddon-3.0.lua:346
 checkActionCooldown = <function> defined @CooldownToGo/CooldownToGo.lua:565
 checkItemCooldown = <function> defined @CooldownToGo/CooldownToGo.lua:622
 RegisterMessage = <function> defined @AtlasLootClassic/Libs/CallbackHandler-1.0/CallbackHandler-1.0.lua:80
 UnregisterMessage = <function> defined @AtlasLootClassic/Libs/CallbackHandler-1.0/CallbackHandler-1.0.lua:143
 hooks = <table> {
 }
 updateCooldown = <function> defined @CooldownToGo/CooldownToGo.lua:669
 showCooldown = <function> defined @CooldownToGo/CooldownToGo.lua:544
 db = <table> {
 }
 SetEnabledState = <function> defined @Attune/Libs/AceAddon-3.0/AceAddon-3.0.lua:420
 Hook = <function> defined @Bartender4/libs/AceHook-3.0/AceHook-3.0.lua:273
 enabledState = true
 checkContainerItemCooldown = <function> defined @CooldownToGo/CooldownToGo.lua:616
 RegisterEvent = <function> defined @AtlasLootClassic/Libs/CallbackHandler-1.0/CallbackHandler-1.0.lua:80
 frameBG = CDTGFrameBG {
 }
 toggleLocked = <function> defined @CooldownToGo/CooldownToGo.lua:745
 profiles = Frame {
 }
 IterateModules = <function> defined @Attune/Libs/AceAddon-3.0/AceAddon-3.0.lua:433
 RawHook = <function> defined @Bartender4/libs/AceHook-3.0/AceHook-3.0.lua:312
 frame = CooldownToGoFrame {
 }
 AppName = "CooldownToGo"
 applyFontSettings = <function> defined @CooldownToGo/CooldownToGo.lua:305
 updateIgnoreListOptions = <function> defined @CooldownToGo_Options/Options.lua:318
 OnInitialize = <function> defined @CooldownToGo/CooldownToGo.lua:369
 UnregisterAllEvents = <function> defined @AtlasLootClassic/Libs/CallbackHandler-1.0/CallbackHandler-1.0.lua:164
 ignoreListOpts = Frame {
 }
 GetModule = <function> defined @Attune/Libs/AceAddon-3.0/AceAddon-3.0.lua:206
 lock = <function> defined @CooldownToGo/CooldownToGo.lua:351
 setOption = <function> defined @CooldownToGo_Options/Options.lua:253
 defaultModuleLibraries = <table> {
 }
 OnUpdate = <function> defined @CooldownToGo/CooldownToGo.lua:442
 applySettings = <function> defined @CooldownToGo/CooldownToGo.lua:323
 setIgnoredState = <function> defined @CooldownToGo/CooldownToGo.lua:706
 text = CDTGText {
 }
 GetName = <function> defined @Attune/Libs/AceAddon-3.0/AceAddon-3.0.lua:275
 opts = Frame {
 }
 isMoving = false
 version = "CooldownToGo-v3.0.6"
 removeIgnored = <function> defined @CooldownToGo_Options/Options.lua:328
 setColor```

**Comment by mitchnull** [comment]
should be fixed in v3.0.8, please check.

**Comment by Nenos-Hu** [comment]
Yep, it's fixed. Thanks!
